mockwebserver/src/main/java/com/squareup/okhttp/mockwebserver/MockResponse.java

```diff
@@ -164,6 +166,26 @@ public MockResponse setChunkedBody(Buffer body, int maxChunkSize) {
     return this;
   }
 
+    /**
+     * Provide a list of chunks to be streamed back to the client.
+     *
+     * @param chunks
+     * @return
+     */
+  public MockResponse setChunkedBody(List<String> chunks) {
+    removeHeader("Content-Length");
+    headers.add(CHUNKED_BODY_HEADER);
+      Buffer bytesOut = new Buffer();
+    for (String s : chunks) {
+        bytesOut.writeUtf8(Integer.toHexString(s.getBytes(Charset.forName("UTF-8")).length));
```

Comment:
Please add two methods, one that takes a `List<String>` and another that takes a `List<ByteString>`, and rewrite one to call the other. Then we can also rewrite the overload on line 151 to call through to the method that takes a `List<ByteString>`, and only have _one_ place where we call `Integer.toHexString()`.